cpp/log/log_signer.cc

```diff
@@ -142,6 +143,7 @@ LogSigner::SignResult LogSigner::SignV1TreeHead(uint64_t timestamp,
 }
 
 LogSigner::SignResult LogSigner::SignTreeHead(SignedTreeHead* sth) const {
+  LOG(INFO) << "Starting SignTreeHead for size: " << sth->tree_size();
```

Comment:
Actually yes. I have thought that some more info will be useful. I can remove these changes, if you think it is not needed.